### ISSUE TITLE
missing sox extension when using augmentations v2.3.1

### ISSUE THREAD

**Issue by chrizzlemadizzle** [acids-ircam/RAVE#276]
When trying to use `--augment compress` or `--augment gain` in training these errors pop up:
`OSError: libsox.so: cannot open shared object file: No such file or directory`
`RuntimeError: apply_effects_tensor requires sox extension which is not available.`
It doesn't happen when using `--augment mute`
I'm on v2.3.1 using `--config v3` and `--config wasserstein`

**Comment by domkirke** [comment]
yes, you have to install sox beforehand (torchaudio usds the sox backend). Check on the internet ;)
